src/MCPClient/lib/settings/common.py

```diff
@@ -38,8 +38,9 @@
     'elasticsearch_server': {'section': 'MCPClient', 'option': 'elasticsearchServer', 'type': 'string'},
     'elasticsearch_timeout': {'section': 'MCPClient', 'option': 'elasticsearchTimeout', 'type': 'float'},
     'search_enabled': [
-        {'section': 'MCPClient', 'option': 'disableElasticsearchIndexing', 'type': 'iboolean'},
```

Comment:
This was on purpose, now I remember. For backward compatibility reasons. We made a release at some point where `disableElasticsearchIndexing` was the name of the attribute so now we're committed to support it. This was only true in MCPClient, that's why you don't see it in other components.